src/pages/edge/adobe-journey-optimizer-decisioning/release-notes.md

```diff
@@ -8,39 +8,49 @@ keywords:
 
 # Release notes
 
+## February 6, 2025
+
+### Adobe Journey Optimizer - Decisioning extension 1.1.0
+
+Added configuration capability to the Adobe Journey Optimizer - Decisioning extension in the Launch UI for mobile Tag Properties.
```

Comment:
Unfortunately, it's not called Launch any more - so this has to be (and I'm so sorry for how stupid this sounds) ".... in the Tags UI for mobile Tag Properties.".